saleor/core/utils/lazyobjects.py

```diff
@@ -0,0 +1,39 @@
+import functools
+from typing import Any, Callable, Optional
+
+from django.utils.functional import LazyObject, SimpleLazyObject, empty
+
+
+def lazy_no_retry(func: Callable) -> SimpleLazyObject:
+    """Wrap SimpleLazyObject while ensuring it is never re-evaluated on failure.
+
+    Wraps a given function into a ``SimpleLazyObject`` class while ensuring
```

Comment:
Nitpick, but why use double backticks if you're not quoting code that contains a backtick?

Reply:
Not sure what's the convention in Saleor core for comments. iirc some places uses the ReStructuredText syntax thus double back-tick would be the way of doing a inline code otherwise it would be a reference.